src/main/java/net/datafaker/transformations/JsonTransformer.java

```diff
@@ -157,17 +169,7 @@ private static Map<Character, String> createEscapeMap() {
             Map.entry('\u0005', "\\u0005"),
             Map.entry('\u0006', "\\u0006"),
             Map.entry('\u0007', "\\u0007"),
-            // map.put('\u0008', "\\u0008");
-            // covered by map.put('\b', "\\b");
-            // map.put('\u0009', "\\u0009");
-            // covered by map.put('\t', "\\t");
-            // map.put((char) 10, "\\u000A");
-            // covered by map.put('\n', "\\n");
             Map.entry('\u000B', "\\u000B"),
-            // map.put('\u000C', "\\u000C");
-            // covered by map.put('\f', "\\f");
-            // map.put((char) 13, "\\u000D");
-            // covered by map.put('\r', "\\r");
```

Comment:
I don't think we need to remove it
this clarifies why we don't have `map.put('\u0008', "\\u0008");` and thers

otherwise each time this question will pop up

Reply:
Reverting to the previous state.

Wouldn't be better to add this as a comment in the method instead of commented code?
Something like this:

```java
    /*
     * The following entries are not added to the map because they are covered by another existing one
     *
     * map.put('\u0008', "\\u0008"); -> covered by map.put('\b', "\\b");
     * map.put('\u0009', "\\u0009"); -> covered by map.put('\t', "\\t");
     * map.put((char) 10, "\\u000A"); -> covered by map.put('\n', "\\n");
     * map.put('\u000C', "\\u000C"); -> covered by map.put('\f', "\\f");
     * map.put((char) 13, "\\u000D"); -> covered by map.put('\r', "\\r");
     */
```